content/1.7/back-office/shipping/carriers/Add-edit-carrier.md

```diff
@@ -0,0 +1,214 @@
+
+If the total weight or total price of the cart is not in one of the defined ranges and if the chosen option is "Disable carrier", then the carrier will be disabled.
+If the total weight or total price of the cart is not in one of the defined ranges and if the chosen option is "Apply the cost of the highest defined range", then the carrier price will be the one corresponding to the highest range.
+
+ - **Ranges** 
+ 2 fields are displayed:
+    - Will be applied when the _weight_ or _price_ is >= _weight_ or _currency_
+    - Will be applied when the _weight_ or _price_ is < _weight_ or _currency_ 
+    The weight or the currency is displayed depending on the option chosen above for the billing. By default, this field is empty
+Only numbers can be entered in the 2 ranges fields above. If one of the fields contains something other than a number the following error message is displayed: "This range is not valid" and the fields for all zones are disabled.
+If one of the ranges overlaps another, the following error message is displayed: "Ranges are overlapping" and the fields for all zones are disabled.
+    - **All** This field is disabled if the checkbox isn't checked. When the checkbox is checked, the field is enabled and all the zones below become checked
+ 
+All the zones existing in International > Locations > Zones are displayed below. The disabled zones are marked as inactive. 
+Each zone has a checkbox allowing to enable its field.
+Only numbers are allowed in zones fields, inputting any other symbol will result in the field being marked in red. 
+
+![Ranges](/img/Ranges.png)
+
+## Call to action 
+
+There is a button "Add new range" below the table. When this button is pressed, a new column is displayed. When clicking on it and if one of the existing ranges isn't completed, the following message is displayed "Please validate the last range before creating a new one." 
+Under each range added, a "Delete" button is displayed. When clicking on it, a modal is displayed: "Are you sure to delete this range?". The range is deleted after clicking on "ok".
+
+  - **Previous.** In the first step, this button is disabled. After clicking on it, the previous step is displayed.
+  - **Next.** If there is an error, this button is disabled. After clicking on it, the next step is displayed.
+  If "free shipping" is disabled and if no zone is selected, the following error message is displayed: "Please select at least one zone".
+  - **Finish.** If there is an error, this button is disabled. After clicking on it, the carrier is saved and the [carriers listing]({{<ref "carriers-listing.md">}} "Carriers listing") page is displayed with the following success message: "Successful creation" if the carrier was added or "Successful update" if the carrier was edited.
+
+# Size, weight, and group access (4th step)
```

Comment:
Both call to action logo and steps are described at the begining. Could you please remove them from this section?